.changes/next-release/Bug Fix-3c1e992b-b24d-418d-a91b-a0cf4d2e7b66.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Static analysis of document symbols used as context in Q requests must filter tree-sitter \"extra\" CST nodes (comments) in order to avoid crashes which freeze the Q chat panel"
```

Comment:
```suggestion
	"description": "Amazon Q: Static analysis of document symbols used as context in Amazon Q requests must filter tree-sitter \"extra\" CST nodes (comments) in order to avoid crashes which freeze the chat panel"
```